variables.env

```diff
@@ -153,10 +159,17 @@ WIKIBASEMANIFEST_COMMIT=8c7cdae4d5a1e68b19fcca8eb5c2a7b6e3dc3e55
 WIKIBASELOCALMEDIA_COMMIT=74c3f70893ffd76feefa9f1e3d77052db20ec3ac
 # https://github.com/ProfessionalWiki/WikibaseEdtf/commits/master
 WIKIBASEEDTF_COMMIT=5596403a27d8e4a5ac044257c49be3e0685318a6
+
+
+# ##############################################################################
+# Quickstatements
+# ##############################################################################
+# Updated automatically by ./build.sh update_hashes
 # https://github.com/magnusmanske/quickstatements/commits/master
 QUICKSTATEMENTS_COMMIT=c4b2c6b086b319aa32dcdd7a323edf188faaa66d
 # https://bitbucket.org/magnusmanske/magnustools/commits/branch/master
 MAGNUSTOOLS_COMMIT=5b8cea412000072a2c8753023c11472a4ac11ef5
+QUICKSTATEMENTS_VERSION=$WMDE_RELEASE_VERSION
```

Comment:
Even though they will all be the same for now, I'd go ahead and make all of these there literal version number (e.g. 1.0.0, etc) vs reusing the WMDE Version Variable, as they will diverge going forward.